### PR TITLE
hide mhr tile, update to assets tile

### PR DESCRIPTION
**Pull request by kialj876**
https://github.com/bcgov/entity/issues/15396

- changes needed to set hidden=false in prod for mhr product

### CODE REVIEW

File: pages/dashboard.vue [hunk 1]
```diff
@@ -152,26 +153,35 @@ export default Vue.extend ({
...
-          if (thisProduct.title !== 'placeholder_title') {
-            this.subscribedProducts.push(thisProduct)
+        let thisProduct = getProductInfo(this.$config, currentProducts[i].code)
+
+        if (thisProduct.title === 'placeholder_title') {
+          console.error('Product tile not implemented:', currentProducts[i].code)
+          continue
+        }
+
+        // block for MHR/PPR special rules
```
Comment (kialj876) [inline pages/dashboard.vue:173]: all mhr/ppr special logic contained in one place now (thought this made it a little clearer)

File: pages/dashboard.vue [hunk 2]
```diff
@@ -152,26 +153,35 @@ export default Vue.extend ({
...
         }
-        const thisProduct = getProductInfo(this.$config, currentProducts[i].code)
 
-        // if user has both MHR and PPR product codes - don't add the tiles for them
-        if (!isMhrPpr || (currentProducts[i].code !== ProductCode.MHR && currentProducts[i].code !== ProductCode.PPR)) {
-          if (thisProduct.title !== 'placeholder_title') {
-            this.subscribedProducts.push(thisProduct)
+        let thisProduct = getProductInfo(this.$config, currentProducts[i].code)
+
+        if (thisProduct.title === 'placeholder_title') {
```
Comment (kialj876) [inline pages/dashboard.vue:168]: this check used to be at the bottom -- thought it was a bit clearer to do this right away and added error log

File: pages/dashboard.vue [hunk 3]
```diff
@@ -152,26 +153,35 @@ export default Vue.extend ({
...
       )
-      const isMhrPpr = hasMhrAndPprProducts(currentProducts)
 
       // only show products with no placeholder
       for (let i = 0; i < currentProducts.length; i++) {
-        if (currentProducts[i].code === ProductCode.BUSINESS_SEARCH) {
-          if (!getFeatureFlag('bcregistry-ui-bus-search-enabled')) continue
+        const enabledFF = `bcregistry-ui-${currentProducts[i].code}-enabled`
+        // ensure has a default (otherwise it might not have an enabled flag set in LD)
+        if (hasDefaultValue(enabledFF) && !getFeatureFlag(enabledFF)) {
```
Comment (kialj876) [inline pages/dashboard.vue:161]: now future products we don't want visible just need the flag and the default (no specific code logic)

**Review by cameron-eyds — APPROVED** [review]
LGTM!

**Review by doug-lovett — APPROVED** [review]
Thanks Kial, looks good.

**Pull request merged**